roles/openshift_metering/files/metering-helm-operator-deployment.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: metering-helm-operator
+  labels:
+    app: metering-helm-operator
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app: metering-helm-operator
+  template:
+    metadata:
+      labels:
+        app: metering-helm-operator
+    spec:
+      securityContext:
+        runAsNonRoot: true
+      containers:
+      - name: metering-helm-operator
+        image: "quay.io/coreos/chargeback-helm-operator:0.6.1-latest"
+        args: ["run-operator.sh"]
+        imagePullPolicy: Always
```

Comment:
Would `IfNotPresent` work?

Reply:
It probably would, however this is what we've been using for a while without trouble. I prefer Always as you can be sure if an image is updated you're not going to see inconsistencies, but we only mutate non-release tags anyways.